src/sys/inotify.rs

```diff
@@ -5,17 +5,17 @@
 //!
 //! Inotify is a Linux-only API to monitor filesystems events.
 //!
-//! For more documentation, please read [the inotify(7) man page.](http://man7.org/linux/man-pages/man7/inotify.7.html)
+//! For more documentation, please read [inotify(7).](http://man7.org/linux/man-pages/man7/inotify.7.html)
 //! 
 //! Example
 //! =======
 //!
 //! * Monitor all events happening in directory "test":
 //! ```rust,no_run
-//! use nix::sys::inotify::{EventFlags,InotifyInitFlags,InotifyInstance}
+//! use nix::sys::inotify::{EventFlags,InotifyInitFlags,Inotify};
 //!
 //! // We create a new inotify instance.
-//! let instance = InotifyInstance::init(InotifyInitFlags::empty())
+//! let instance = Inotify::init(InotifyInitFlags::empty())
 //!     .expect("inotify_init failed");
```

Comment:
For the doc tests, you can replace the `.expect` with a simple `.unwrap`.  It gets the point across.